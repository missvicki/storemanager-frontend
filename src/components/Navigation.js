import React from 'react';
import { Menu, Header} from 'semantic-ui-react';
import { NavLink } from 'react-router-dom';

const Navigation = (props) => {
    return (
        <div>
            <Menu secondary style={{ background: "teal" }}>
                <Menu.Item header style={{ margin: "0 auto" }}>
                    <NavLink to="/">
                        <Header as="h1" textAlign="center" padded="very" style={{ color: "white" }}>Store Manager</Header>
                    </NavLink>
                </Menu.Item>
                <Menu.Menu position='right' style={{ margin: "0 auto" }}>
                    <Menu.Item
                        style={{ color: "white", textSize: "3em" }}
                        name='logout'
                        onClick={props.onLogoutClick}
                    />
                </Menu.Menu>
            </Menu>
        </div>
    );
};

export default Navigation;