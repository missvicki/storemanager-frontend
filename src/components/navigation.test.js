import React from 'react';
import { shallow } from 'enzyme';

import Navigation from './Navigation';

describe('Navigation', () => {
    const props = {
        onLogoutClick: () => {}
    };
    
    const nav = shallow(<Navigation {...props}/>);
    it('should render the Navigation Component', () => {
        expect(nav).toMatchSnapshot();
    });
});
