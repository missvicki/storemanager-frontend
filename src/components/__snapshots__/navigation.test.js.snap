// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation should render the Navigation Component 1`] = `
<div>
  <Menu
    secondary={true}
    style={
      Object {
        "background": "teal",
      }
    }
  >
    <MenuItem
      header={true}
      style={
        Object {
          "margin": "0 auto",
        }
      }
    >
      <NavLink
        to="/"
      >
        <Header
          as="h1"
          padded="very"
          style={
            Object {
              "color": "white",
            }
          }
          textAlign="center"
        >
          Store Manager
        </Header>
      </NavLink>
    </MenuItem>
    <MenuMenu
      position="right"
      style={
        Object {
          "margin": "0 auto",
        }
      }
    >
      <MenuItem
        name="logout"
        onClick={[Function]}
        style={
          Object {
            "color": "white",
            "textSize": "3em",
          }
        }
      />
    </MenuMenu>
  </Menu>
</div>
`;
